### ISSUE TITLE
listen requires Ruby version

### ISSUE THREAD

**Issue by TamaHobbit** [jekyll/jekyll#5426]
- [x] I believe this to be a bug, not a question about using Jekyll.
- [ ] I updated to the latest Jekyll (or) if on GitHub Pages to the latest `github-pages`
- [ ] I read the CONTRIBUTION file at https://jekyllrb.com/docs/contributing/
- [ ] This is a feature request.

---

- [ ] I am on (or have tested on) ***macOS*** 10+
- [x] I am on (or have tested on) ***Debian/Ubuntu*** GNU/Linux
- [ ] I am on (or have tested on) ***Fedora*** GNU/Linux
- [ ] I am on (or have tested on) ***Arch*** GNU/Linux
- [ ] I am on (or have tested on) ***Other*** GNU/Linux
- [ ] I am on (or have tested on) ***Windows*** 10+

---

- [x] I was trying to install.
- [ ] There is a broken Plugin API.
- [ ] I had an error on GitHub Pages, and I have reproduced it locally.
- [ ] I had an error on GitHub Pages, and GitHub Support said it was a Jekyll Bug.
- [ ] I had an error on GitHub Pages and I did not test it locally.
- [ ] I was trying to build.
- [ ] It was another bug.

## My Reproduction Steps

```
$ sudo gem install jekyll                                                                                                           
Fetching: liquid-3.0.6.gem (100%)
Fetching: kramdown-1.12.0.gem (100%)
Fetching: mercenary-0.3.6.gem (100%)
Fetching: safe_yaml-1.0.4.gem (100%)
Fetching: colorator-1.1.0.gem (100%)
Fetching: rouge-1.11.1.gem (100%)
Fetching: sass-3.4.22.gem (100%)
Fetching: jekyll-sass-converter-1.4.0.gem (100%)
Fetching: rb-fsevent-0.9.7.gem (100%)
Fetching: ffi-1.9.14.gem (100%)
Building native extensions.  This could take a while...
Fetching: rb-inotify-0.9.7.gem (100%)
Fetching: listen-3.0.8.gem (100%)
ERROR:  Error installing jekyll:
        listen requires Ruby version >= 1.9.3.

$ ruby --version
ruby 2.1.5p273 (2014-11-13) [x86_64-linux-gnu]
```

**Comment by TamaHobbit** [comment]
Looks like I have lots of versions of ruby installed. Still, the issue is that your installation page says installing jekyll is supposed to be easy. I've scarcely moved past helloWorld in ruby, so this is definitely inside the black box for me.

$ dpkg --get-selections | grep -i ruby
libruby1.8                                      install
libruby2.1:amd64                                install
ruby                                            install
ruby-debian                                     install
ruby-dev                                        install
ruby-gettext                                    install
ruby-httpclient                                 install
ruby-locale                                     install
ruby-soap4r                                     install
ruby-text                                       install
ruby-unicode                                    install
ruby-xmlparser                                  install
ruby1.8                                         install
ruby1.8-dev                                     install
ruby2.1                                         install
ruby2.1-dev:amd64                               install
rubygems                                        install
rubygems-integration                            install

**Comment by envygeeks** [comment]
@TamaHobbit this belongs to Listen, and RubyGems, we can't fix their bugs.

**Comment by envygeeks** [comment]
Let me rephrase that, ***we don't fix their bugs***

**Comment by TamaHobbit** [comment]
"The best way to install Jekyll is via RubyGems"

What's the second best way to install Jekyll, then?

**Comment by envygeeks** [comment]
There are really only two ways to install it, via Gemfile, and via RubyGems, both will probably exhibit the same bug, but Bundler might not so try adding Jekyll to a gemfile and do bundle install and see if you get the same problem, if you don't then it's probably a RubyGem bug.  You can also try updating your RubyGems to see if they already solved the bug too.

**Comment by TamaHobbit** [comment]
Ok. Thanks for your time.

It sounds like that's going to take me hours to figure out what to update, and may put me in dependency hell, so I'll just wait until those bugfixes have made their way into debian's packages.

Listen says the problem is with Ruby 2.1, and specifies a workaround, but it's greek to me. Maybe the next reader can use this. 
https://github.com/guard/listen/wiki/Ruby-version-requirements
